solc/CommandLineInterface.cpp

```diff
@@ -877,11 +877,13 @@ void CommandLineInterface::handleCombinedJSON()
 	if (m_options.compiler.combinedJsonRequests->ast)
 	{
 		output[g_strSources] = Json::Value(Json::objectValue);
+		unsigned sourceIndex = 0;
 		for (auto const& sourceCode: m_fileReader.sourceUnits())
 		{
 			ASTJsonExporter converter(m_compiler->state(), m_compiler->sourceIndices());
 			output[g_strSources][sourceCode.first] = Json::Value(Json::objectValue);
-			output[g_strSources][sourceCode.first]["AST"] = converter.toJson(m_compiler->ast(sourceCode.first));
+			output[g_strSources][sourceCode.first]["ast"] = converter.toJson(m_compiler->ast(sourceCode.first));
```

Comment:
Why do you change this? We can't just change the output structure of combined-json like this :-).

Reply:
it looks like that this is done differently in `StandardCompiler` see https://github.com/ethereum/solidity/pull/14171#discussion_r1180855723 - I did this to just have the same output format for the import/export tests. However, I can just change the import/export tests accordingly - but it's a bit ugly then 😅